src/patches/Briefing.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using BattleTech;
+using BattleTech.Data;
+using BattleTech.Rendering;
+using BattleTech.Rendering.Mood;
+using BattleTech.UI;
+using Harmony;
+using HBS;
+using HBS.Collections;
+
+namespace EnvironmentalDesignMasks
+{
```

Comment:
Use sameline braces, same as the rest of the repo. Always best to follow a given repo's existing styles - I do the same when working in IRTweaks.